docs/sources/plugins/catalog.md

```diff
@@ -6,17 +6,59 @@ weight = 1
 
 # Plugin catalog
 
-Catalog for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the Marketplace.
+The Plugin catalog, new in Grafana 8.0, allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the catalog.
 
-[screenshot placeholder]
+<div class="medium-6 columns">
+  <video width="800" height="500" controls>
+    <source src="/assets/videos/plugins-catalog-install-8-0.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+  </video>
+</div>
 
-To use the Catalog for Grafana, you first need to enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#catalog_app_enabled" >}}).
+Before you can use the Plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.
```

Comment:
@achatterjee-grafana, what are your thoughts on capitalization here? I would err on the side of not capitalizing it in most cases. Here, it could be read as a generic plugin catalog.

Reply:
Agree, we should keep this as "Catalog".

Reply:
Maybe "plugin catalog" for generic mentions and "Catalog" when you're talking about the specific screen? Not sure we want to necessarily capitalize every use of catalog, either. That would get annoying. I'm trying to come up with a coherent rule we could add to the style guide and drawing a blank.

Reply:
Thinking over, let us stick to "plugin catalog".